### PR TITLE
Chapter 13 Outline

### PR DESCRIPTION
**Pull request by ian-flores**
This PR includes the outline for chapter 13. In my opinion it is a little bit short so if people want to chime in with ideas I'm more than open to include them.

### CODE REVIEW

File: chapters/13-package-release.Rmd [hunk 1]
```diff
@@ -5,15 +5,33 @@
...
-> `rhub::validate_email("youremail@email.com")`
-
-TODO: This question would be better suited as a core exercise, not in the final exercise
-1. Read the usage limits on GitHub Actions and rhub
-> - https://docs.github.com/en/actions/reference/usage-limits-billing-and-administration
-> - https://docs.r-hub.io/#regularly
-TODO: Add a question to assess whether they've read it?
+## Learning Objectives
+
+- Describe the importance of using a build system for validating softwate
```
Comment (lwjohnst86) [inline chapters/13-package-release.Rmd:10]:
```suggestion
- Describe the importance of using a build system for validating software and how to apply it
```

File: chapters/13-package-release.Rmd [hunk 2]
```diff
@@ -5,15 +5,33 @@
...
+- Identify ethical guidelines when using build systems like RHub
```
Comment (lwjohnst86) [inline chapters/13-package-release.Rmd:11]:
```suggestion
- Understand and implement ethical guidelines when using build systems like RHub
```

**Review by lwjohnst86 — APPROVED** [review]
I think it looks good. I can't think of anything else to add, maybe when we get to the next review phases?

**Review by cwickham — APPROVED** [review]
This looks good to me too.  Only thought, should we talk about `use_github_release()` or `use_release_issue()` (and any of the contents of that checklist)?

File: chapters/13-package-release.Rmd [hunk 3]
```diff
@@ -5,15 +5,33 @@
...
+    * Text: Talk about why it is important to check in multiple environments
+        * Text: Cover dependencies, file management, build processes
+    * Exercise: Create an account in RHub and validate your email
+        * `rhub::validate_email("youremail@email.com")`
+    * Text: Talk about behaviour when using RHub and other build systems
+        Exercise: Read https://docs.r-hub.io/ and answer "Can you use RHub to continously build and deploy your package?"
+
+* GitHub Release Process
+    * Text: Talk about Git tagging
+    * Text: Talk about GitHub releases
```
Comment (lwjohnst86) [inline chapters/13-package-release.Rmd:28]:
```suggestion
    * Text: Talk about GitHub releases
    * Code: use_github_release(), use_release_issue()
```

File: chapters/13-package-release.Rmd [hunk 4]
```diff
@@ -5,15 +5,33 @@
...
+    * Exercise: Perform a GitHub release of the core package
+
+* Version management
+    * Text: When should I upgrade the package version?
```
Comment (lwjohnst86) [inline chapters/13-package-release.Rmd:32]:
```suggestion
    * Text: When should I upgrade the package version?
    * Code: use_version(), use_dev_version()
```

**Review by lwjohnst86 — APPROVED** [review]
Just adding text based on Charlotte's comments.

**Pull request merged**